.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: python
+python:
+  - 3.6
+services:
+  - postgresql
+addons:
+  chrome: stable
+cache:
+  directories:
+    - $HOME/.cache/pip
+    - $HOME/.yarn-cache
+    - node_modules
+before_install:
+  - nvm install 10
+  - curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+  - echo "deb http://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+  - sudo apt-get update -qq
+  - sudo apt-get install -y -qq yarn
+  - google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &
+install:
+  - pip install cookiecutter
+  - pip install pip-tools
+  - yarn global add @angular/cli
+script:
+  - cookiecutter . --no-input
+  - cd dighum_pro
+  - cat bootstrap.log
+  - cd backend && pip-compile && cd ..
+  - cd functional-tests && pip-compile && cd ..
+  - yarn
+  - yarn django migrate
```

Comment:
This is a work-around for the created virtualenv being in Python 2 for some reason:

> /home/travis/build/UUDigitalHumanitieslab/cookiecutter-webapp-deluxe/dighum_pro/.env/local/lib/python2.7/site-packages/pip/_vendor/urllib3/util/ssl_.py:354: SNIMissingWarning: An HTTPS request has been made, but the SNI (Server Name Indication) extension to TLS is not available on this platform. This may cause the server to present an incorrect TLS certificate, which can cause validation failures. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
>   SNIMissingWarning

https://travis-ci.com/UUDigitalHumanitieslab/cookiecutter-webapp-deluxe/builds/108120856

The same also happened on my Linux machine (using a Python 3 virtualenv). So I added this workaround to use the virtualenv created by Travis itself which is using Python 3.

Reply:
This is probably the builtin virtualenv of the OS image. `pip install virtualenv` before starting the cookiecutter might resolve this.